mrp_bom_location/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+#. Go to *Manufacturing > Bill of Materials*.
+#. Pick or create one of them.
+#. You will see a new field to fill called "Location".
+#. One the structure report *BOM > Print > BOM Structure* location field is present
```

Comment:
Typo:
On the structure report *BOM > Print > BOM Structure* location field is present

Reply:
Fixed it.